server/views.py

```diff
@@ -1,5 +1,38 @@
-from django.shortcuts import render
-
+from django.shortcuts import render, redirect
+from .forms import UserForm, PlayerForm, MembershipForm, VaccinationForm
+from .models import User
 
 def home(request):
-    return render(request, "index.html")
+    return render(request, 'index.html')
+
+def registration_view(request):
+    if request.method == 'POST':
+        user_form = UserForm(request.POST)
+        player_form = PlayerForm(request.POST)
+        membership_form = MembershipForm(request.POST)
+        vaccination_form = VaccinationForm(request.POST)
+
+        if user_form.is_valid() and player_form.is_valid() and membership_form.is_valid() and vaccination_form.is_valid():
+            user = user_form.save(commit=False)
+            player = player_form.save(commit=False)
+            membership = membership_form.save(commit=False)
+            vaccination = vaccination_form.save(commit=False)
```

Comment:
`commit=False` wouldn't save them to the DB, right?

Reply:
yes, changed it